### DISCUSSION TITLE
Restore previous import

### DISCUSSION THREAD

**Discussion by Phantomx2** [immich-app/immich#7228] [Q&A]
Hi,
long story short, I lost the database for Immich.
I still have all the pictures on my server in the Immich folder ex: Pictures/library/admin/...

I cannot move or delete them as they are read only now. What would be the best way to reimport everything as is in Immich?

Thanks!

**Comment by mmomjian** [comment]
Are you sure the database is lost? You can start a new installation of Immich and upload the images from that folder using the command line uploader.

**Reply by Phantomx2** [reply]
yes I am sure I lost my DB unfortunatly. 
With the CLI it won't duplicate the files? Right now if I drag and drop extisting pictures from the library folder the get duplicated.

**Reply by mmomjian** [reply]
I'm rather confused, but if you lost your database, how are you still seeing any files? Do you have it mounted as an external library?

**Reply by mmomjian** [reply]
Oh, you still have the library folder that I assume you are viewing with a file browser. Best thing to do it start over, import those pictures into Immich with a new library path, and then delete the old library once you are done.

**Reply by Phantomx2** [reply]
I see the files by browsing on the drive, I don't see them in immich.

**Reply by Phantomx2** [reply]
I mean it's kinda fast to reimport, so I think I will just do that indeed. Was just curious to see if there was a better way.

**Reply by mmomjian** [reply]
Yes, I don't think there is a way to re-fresh the library because all of the important details were stored in the database.
